recipes/fortnet/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "fortnet" %}
+{% set version = "0.7.2" %}
+{% set build = 0 %}
+{% set mpi = mpi or "nompi" %}
+{% set origin = "vanderhe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ origin }}/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 77f9477231491af7400ef74061d89149b67eae14bd8fedce690a4a79b2294c9c
+
+{% if mpi == "nompi" %}
+{% set build = build + 100 %}
+{% endif %}
+build:
+  number: {{ build }}
+  skip: True  # [not linux]
+
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
+  string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ build }}"
+
+  run_exports:
+    - {{ name|lower }} * {{ mpi_prefix }}_*
+    - {{ pin_subpackage( name|lower, max_pin='x.x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+    - python
+    - h5py
+    - numpy
+    - cmake
+    - ninja
+    - fypp
+  host:
+    - mpifx * mpi_{{ mpi }}_*  # [mpi != "nompi"]
+    - libblas
+    - hdf5 >=1.10.4
+    - hdf5 * nompi_*
+  run:
+    - {{ pin_compatible('mpifx', max_pin='x.x') }} mpi_{{ mpi }}_*  # [mpi != "nompi"]
+    # hdf5 >=1.10.4 has run_exports
+    - hdf5
+
+test:
+  commands:
+    - export OMPI_MCA_plm=isolated OMPI_MCA_btl_vader_single_copy_mechanism=none OMPI_MCA_rmaps_base_oversubscribe=yes  # [mpi == "openmpi"]
+    - test -f $PREFIX/bin/fnet
+    - $PREFIX/bin/fnet | grep "Error in opening file"  # [mpi == "nompi" and build_platform == target_platform]
```

```suggestion
    - fnet | grep "Error in opening file"  # [mpi == "nompi" and build_platform == target_platform]
```
Shouldn't need to use full path.